db/seeds.rb

```diff
@@ -31,6 +31,9 @@
   Service.create!(:name => service)
 end
 
```

Comment:
Can you pls add a `delete_all` call for `Job` i.e. https://github.com/OperationCode/operationcode_backend/blob/master/db/seeds.rb#L9-L16